securedrop_client/gui/widgets.py

```diff
@@ -260,13 +260,13 @@ def launch(self):
             logger.debug("Deleting source %s" % (self.source.uuid,))
             self.controller.delete_source(self.source)
 
-    def _construct_message(self, source):
+    def _construct_message(self, source: Source) -> None:
```

Comment:
type checking nit: either `None` or `Message` can be returned from this function

(added https://github.com/freedomofpress/securedrop-client/issues/25 to project board for next sprint)